packages/edit-post/src/components/fullscreen-mode/index.js

```diff
@@ -17,11 +18,15 @@ class FullscreenMode extends Component {
 
 	sync() {
 		const { isActive } = this.props;
+		let delay = 0;
 		if ( isActive ) {
 			document.body.classList.add( 'is-fullscreen-mode' );
+			delay = 300; // .is-fullscreen-mode has a fade in animation of 0.3s
```

Comment:
I'm not happy about this solution proposed here. What if CSS overrides animation to finish in 0.4s? Is there any other way to ensure that this event is properly triggered?